docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<p>Is there a reason why we're not using:</p><pre class="ql-syntax" spellcheck="false">%pip install -q dwave_qbsolv
%pip install -q dwave-neal
%pip install -q PyPortfolioOpt
%pip install -q yfinance
</pre><p>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>only because i was under the impression that we should try to <code>python3 -m pip install</code> instead of just doing <code>pip install</code> when we can. is that not the case?</p>

Reply:
As far as user notebooks, other notebooks use `%pip install -q <package>` so I think we should stick to that standard.

Reply:
sounds good - done.